druid/src/event.rs

```diff
@@ -147,6 +147,7 @@ pub enum InternalEvent {
     MouseLeave,
     /// A command still in the process of being dispatched.
     TargetedCommand(Target, Command),
+    RouteTimer(TimerToken, WidgetId),
```

Comment:
Add a comment here describing this variant. Something simple like *Used to route timer events.* would work.